data/mods/Aftershock/items/afs_inactiverobot.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "id": "bot_migoturret",
+    "type": "TOOL",
+    "name": "inactive brain blaster",
+    "description": "This is a horrifying monstrosity.  Using this item is cannibalism in all but taste and involves turning it on and placing it on the ground, where it will attach itself.  If reprogrammed and rewired successfully the turret will identify you as a friendly, and attack all enemies with its revolving laser cannons.",
+    "weight": "1450 g",
+    "volume": "3 L",
+    "price": 600000,
+    "to_hit": -3,
+    "bashing": 8,
+    "material": "alien_resin",
+    "symbol": ";",
+    "color": "green",
+    "use_action": {
+      "type": "place_monster",
+      "monster_id": "afs_mon_migoturret",
+      "difficulty": 6,
+      "moves": 100,
+      "skill1": "cooking",
```

Comment:
Cooking? Is this right?

Reply:
I'm kinda torn on this one and open to suggestions, maybe persuasion, maybe first aid? Cooking has chemistry in it which I think ties into biotech more than computers, electronics or mechanics.

Reply:
I think it is okay for now as cooking also includes chemistry. It could be updated later when we have more skills.